server/WebAPI/DTO/Message.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Http;
+using MimeKit;
+
+namespace WebAPI.DTO
+{
+  public class Message
+  {
+    public List<MailboxAddress> To { get; set; }
+    public string Subject { get; set; }
+    public string Content { get; set; }
+
+    public IFormFileCollection Attachments { get; set; }
```

Comment:
are we going to have attachments for recovery emails?

Reply:
this is a service. there will be no password recovery, but the service is made for the future, maybe someone will need to reset the page in html format, maybe something else will be. I implemented the service as much as possible, implemented everything that could be done in it